src/transformers/utils/__init__.py

```diff
@@ -188,7 +188,7 @@
     is_torch_sdpa_available,
     is_torch_tensorrt_fx_available,
     is_torch_tf32_available,
-    is_torch_tpu_available,
+    is_torch_xla_available,
```

Comment:
We need to leave this as importable whilst it's still going through the deprecation cycle

```suggestion
    is_torch_tpu_available,
    is_torch_xla_available,
```

Reply:
Done